React.podspec

```diff
@@ -1,7 +1,11 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
 Pod::Spec.new do |s|
   s.name                = "React"
-  s.version             = "0.0.1-master"
-  s.summary             = "Build high quality mobile apps using React."
+  s.version             = package['version'].start_with?('0.0.0') ? '0.0.1-master' : package['version']
```

Comment:
We very recently changed (or plan to change) package.json to be version "1000.0.0" in master, so just `package['version']` should suffice